integration/sawtooth_integration/docker/intkey-smoke.yaml

```diff
@@ -49,7 +49,7 @@ services:
     entrypoint: "bash -c \"\
         ./bin/sawtooth admin keygen && \
         ./bin/sawtooth admin genesis && \
-        ./bin/validator -vv\""
```

Comment:
So without the debug logging on it is hard to tell why it has failed. It could be because something in the genesis process isn't right and so the validator never starts building blocks, or it could be something else entirely. The transaction processor logging seems less important, but the validator logging in the test is essential.